reset.css

```diff
@@ -0,0 +1,52 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 

```

Comment:
It's better to put this file inside the `src` folder, maybe in a CSS folder:
Like so: `src/css/reset.css`

Reply:
this reset.css is already in the src foulder, its better to create a css folder and put it in there?

Reply:
the reset.css is now at the same level as `src` and not inside of it, take a look:

![Screen Shot 2022-09-12 at 6 25 30 PM](https://user-images.githubusercontent.com/2808008/189761495-6ac453ed-9f59-4340-81f9-9785596ddea3.png)

but yes the idea is to create a new folder inside `src` called `css`.

Reply:
oh, i miss that